eapeak

```diff
@@ -1,118 +1,138 @@
 #!/usr/bin/env python
+#
+# -*- coding: utf-8 -*-
+#
+#  eapeak
+#
+#  Author: Spencer McIntyre (Steiner) <smcintyre [at] securestate [dot] com>
+#
+#  Copyright 2011 SecureState
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+#  MA 02110-1301, USA.
+#
+#  Shout outs to the SecureState Profiling Team (Thanks Guys!)
+#    agent0x0
+#    f8lerror
+#    jagar
+#    WhIPsmACK0
+#    Zamboni
+#
+#  Additional Thanks To:
+#    Joshua Wright
+#    Zero_Chaos
+#    Steve Ocepek
 
-"""
-	-*- coding: utf-8 -*-
-	eapeak
-	
-	Author: Spencer McIntyre (Steiner) <smcintyre [at] securestate [dot] com>
-	
-	Copyright 2011 SecureState
-	
-	This program is free software; you can redistribute it and/or modify
-	it under the terms of the GNU General Public License as published by
-	the Free Software Foundation; either version 2 of the License, or
-	(at your option) any later version.
-	
-	This program is distributed in the hope that it will be useful,
-	but WITHOUT ANY WARRANTY; without even the implied warranty of
-	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-	GNU General Public License for more details.
-	
-	You should have received a copy of the GNU General Public License
-	along with this program; if not, write to the Free Software
-	Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
-	MA 02110-1301, USA.
-	
-	Shout outs to the SecureState Profiling Team (Thanks Guys!)
-	agent0x0
-	f8lerror
-	jagar
-	WhIPsmACK0
-	Zamboni
-	
-	Additional Thanks To:
-	Joshua Wright
-	
-	Zero_Chaos
-	Steve Ocepek
-		
-"""
+# native imports
+import argparse
+try:
+	import curses # pylint: disable=unused-import
+	curses_enabled = True
+except ImportError:
+	curses_enabled = False
+import logging
+import os
+from select import error as select_error
+from socket import error as socketError	# needed for the error
+import sys
+import thread
+
+# project imports
+from eapeak.parse import CursesEapeakParsingEngine # pylint: disable=no-name-in-module
+try:																	# default to the systems libraries
+	from eapeak.parse import EapeakParsingEngine						# revert to ./lib on fail for development
+except ImportError:
+	sys.path.append(os.getcwd() + os.sep + 'lib')
```

Comment:
Instead of using `os.getcwd()` you should use `__file__`. This is because it's more likely that the user may be in a different directory than the source file. Also instead of concatenating the paths, this should use `os.path.join`, something like `sys.path.append(os.path.join(os.path.dirname(__file__), 'lib'))`.